src/data/services/EcommerceApiService.js

```diff
@@ -47,6 +47,21 @@ class EcommerceApiService {
     return apiClient.post(url, options, 'json');
   }
 
+  static sendLicenseRevoke(licenseId, options) {
```

Comment:
I think this should be in https://github.com/edx/frontend-app-admin-portal/blob/master/src/components/subscriptions/data/service.js instead of EcommerceApiService.